scripts/ChatterboxGetSpeaker/ChatterboxGetSpeaker.gml

```diff
@@ -0,0 +1,27 @@
+/// Returns the string behind the first colon in a line of dialogue, excluding the speaker data if there's any.
+///
+/// @param chatterbox
+/// @param contentIndex
+
+function ChatterboxGetSpeaker(_chatterbox, _index)
+{
+    var _str = ChatterboxGetContent(_chatterbox, _index);
+    if (_str != undefined)
+    {
+        var _colon = string_pos(":", _str);
+        if (!_colon)
+        {
+            show_debug_message("Chatterbox: No speaker found.") return "";
```

Comment:
Please use `__ChatterboxTrace()` to output messages. Additionally, I'm unsure if it's more suitable to return `""` or `undefined` here. Thoughts?

Reply:
I think some people might want certain lines of text with an empty speaker, and returning `undefined` might force them to either add a single colon at the start of a line, which looks a bit ugly and wouldn't make much sense for some writers in my opinion, or code it so if the string is `"undefined"`, the speaker and/or namebox isn't drawn, but they could already do that with just `""`

Reply:
`""` works for me then